.claude/commands/suba.md

```
@@ -0,0 +1,29 @@
+---
+description: /suba Alias - Dual Consultant Validation Sweep
+type: llm-orchestration
+execution_mode: immediate
+---
+## ⚡ EXECUTION INSTRUCTIONS FOR CLAUDE
+**When this command is invoked, YOU (Claude) must execute these steps immediately:**
+**This is NOT documentation - these are COMMANDS to execute right now.**
+**Use TodoWrite to track progress through multi-phase workflows.**
+
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)